src/plugin/negativeYear/index.js

```diff
@@ -0,0 +1,21 @@
+export default (_, c, dayjs) => {
+  const proto = c.prototype
+
+  const parseDate = (cfg) => {
+    const { date } = cfg
+    if (!date) return new Date()
+    const newDate = new Date(date)
+    const fullYear = newDate.getFullYear()
+    if (typeof date === 'string' && date.indexOf(`-${fullYear}`) !== -1) {
+      return dayjs(newDate).subtract(fullYear * 2, 'year').toDate()
+    }
+    return newDate
```

Comment:
we can return the passed in the original cfg.date directly, without dealing with logic like !date

Reply:
@iamkun Got you